docs/pages/workflow/expo-cli.md

```diff
@@ -237,17 +257,87 @@ This command will be disabled if your project is configured to use `metro` for b
 '$ npx expo prebuild',
 ]} cmdCopy="npx expo prebuild" />
 
-Native source code must be generated before a native app can compile. Expo CLI provides a unique and powerful system called _prebuild_, that generates the native code for your project.
+Native source code must be generated before a native app can compile. Expo CLI provides a unique and powerful system called _prebuild_, that generates the native code for your project. To learn more, read the [Expo Prebuild docs](/workflow/prebuild.md).
 
-**For more information:**
+## Config
 
-- [Prebuild](/workflow/prebuild)
-- [Config plugins](/guides/config-plugins)
-- [Native modules API](/modules/module-api)
+Evaluate the Expo config (**app.json**, or **app.config.js**) by running:
 
-## Telemetry
+<Terminal cmd={[
+'$ npx expo config',
+]} cmdCopy="npx expo config" />
+
+- `--full`: Include all project config data
+- `--json`: Output in JSON format, useful for converting an `app.config.js` to an `app.config.json`.
+- `-t, --type`: [Type of config](#config-type) to show.
+
+### Config Type
+
+There are three different config types that are generated from the Expo config:
+
+- `public`: The manifest file for use with OTA updates, think of this like an `index.html` file's `<head />` element but for native apps.
+- `prebuild`: The config that is used for [Expo Prebuild](/workflow/prebuild) including async modifiers. This is the only time the config is not serializable.
+- `introspect`: A subset of the `prebuild` config that only shows in-memory modifications like `Info.plist` or `AndroidManifest.xml` changes. Learn more about [introspection](/guides/config-plugins#introspection).
+
+## Install
+
+Unlike the web, React Native is not backwards compatible, meaning npm packages often need to be the exact right version for the currently installed copy of `react-native` in your project. Expo CLI provides a best-effort tool for doing this using a list of popular packages and the known working version combinations. Simply use the `install` command as a drop-in replacement for `npm install`:
+
+<Terminal cmd={[
+'$ npx expo install expo-camera',
+]} cmdCopy="npx expo install expo-camera" />
+
+You can add as many packages as you'd like:
+
+<Terminal cmd={[
+'$ npx expo install typescript expo-sms',
+]} cmdCopy="npx expo install typescript expo-sms" />
+
+You can directly pass arguments to the underlying package manager by using the `--` operator:
 
-Expo dev tools collect anonymous data about general usage. This helps us know when a command is safe to deprecate (low usage), or when a feature is not working as expected. Telemetry is completely optional, and you can opt out by using the `EXPO_NO_TELEMETRY=1` environment variable.
+<Terminal cmd={[
+'$ yarn expo install typescript -- -D',
+'# yarn add typescript -D',
+]} cmdCopy="yarn expo install typescript -- -D" />
```

Comment:
I'm curious to know using `yarn`, do you want to showcase that a different package manager can be used?  I think this is clever to demonstrate different package managers as examples 💯